### PR TITLE
replace Github social auth token with configurable API token

### PR DESCRIPTION
**Pull request by Rizziepit**

### CODE REVIEW

**Comment by miltontony** [comment]
:ok_hand: looks good so far

**Comment by Rizziepit** [comment]
Great. I'm updating the login screen and writing tests and then this is done.

**Comment by Rizziepit** [comment]
@miltontony @smn ready for review. Requests to the Github API now use basic auth instead of token auth. There is still a token involved though. `GITHUB_USERNAME` will be the username of a personal account that's a member of Universal Core. `GITHUB_TOKEN` is a personal access token created by that account (somewhere under settings).

Side note: I removed all access tokens in HTML + JS. That's the reason for the `teams_json` view and moving `get_repos` and `get_teams` to `utils`.

File: unicoremc/utils.py [hunk 1]
```diff
@@ -122,3 +126,47 @@ def get_default_docker_cmd(application_type, country):
...
+    the result.
+    """
+    if not refresh:
+        return cache.get('repos')
+    url = urljoin(
+        settings.GITHUB_API, 'repos?type=public&per_page=100&page=%s')
+    pageNum = 1
+    repos = []
+    while True:
+        response = requests.get(url % pageNum)
```
Comment (miltontony) [inline unicoremc/utils.py]: @Rizziepit can you add the auth token here too please. There's a limit of 60 requests for unauthenticated requests

**Comment by miltontony** [comment]
:+1: :star2: apart from that 1 addition of auth to the repos call

File: unicoremc/utils.py (hunk 1 above)
Comment (Rizziepit) [inline unicoremc/utils.py]: Fixed!

**Comment by miltontony** [comment]
:+1:  :shipit:

**Pull request merged**